packages/@okta/vuepress-site/docs/concepts/oauth-openid/index.md

```diff
@@ -272,3 +273,30 @@ app -> client: Response
 -->
 
 For information on how to set up your application to use this flow, see [Implement the Client Credentials Flow](/docs/guides/implement-client-creds/).
+
+### SAML 2.0 Assertion flow
+
+The SAML 2.0 Assertion flow is intended for a client app that wants to use an existing trust relationship without a direct user approval step at the authorization server. It enables a client application to obtain an existing authorization in the form of a valid, signed SAML assertion from a SAML Identity Provider. The client app can then exchange it for an OAuth access token from the OAuth authorization server. For example, this flow is useful when you want to fetch data from APIs that only support delegated permissions without prompting the user for credentials.
```

Comment:
It does not `enables a client application to obtain an existing authorization`, should it be `It enables a client application to obtain an authorization from an existing trust relationship in the form of a valid, signed SAML assertion from a SAML Identity Provider. `